clients/src/test/java/org/apache/kafka/clients/consumer/internals/MembershipManagerImplTest.java

```diff
@@ -1403,6 +1403,14 @@ public void testHeartbeatSentOnStaledMember() {
         assertEquals(MemberState.JOINING, membershipManager.state());
     }
 
+    @Test
+    public void testMemberJoiningTransitionsToStableWhenReceivingEmptyAssignment() {
```

Comment:
Is there also a test for the second case? I skimmed over the test class but could not find one.

Reply:
Yes, there is, `testNewEmptyAssignmentReplacesPreviousOneWaitingOnMetadata` covers the other case.